README.md

```diff
@@ -1 +1,144 @@
+
+5. Wait for a reviewer to approve your pull request. Address any comments as needed.
+
+6. After you receive approval, merge your pull request and delete your working branch.
+
+## Technical review
+
+### Prerequisite
+
+- You must have an article in the `initial_draft` branch.
+
+### Workflow
+
+You will receive technical review during this process. Complete the following steps to get your initial draft into the `technical_review` branch:
+
+1. Create a new working branch. Use `initial_draft` as your **Base**. Name it using the following temple: `<your_initials>_tech`
+
+2. In your working branch, commit any changes you want to your `<article_name>.md` to prepare to tehcnical review.
+
+3. Create a pull request. Use your working branch `<your_initials>_tech` as the **compare** and use `technical_review` as the the **base**.
+
+4. Wait for a reviewer to approve your pull request. Address any comments as needed.
+
+5. After you receive approval, merge your pull request and delete your working branch.
+
+## Instructional review
+
+### Prerequisite
+
+- You must have an article in the `technical_review` branch.
+
+### Workflow
+
+You will receive instructional review during this process. Complete the following steps to get your initial draft into the `instructional_review` branch:
+
+1. Create a new working branch. Use `technical_review` as your **Base**. Name it using the following temple: `<your_initials>_instruct`
+
+2. In your working branch, commit any changes you want to your `<article_name>.md` to prepare for instructional review.
+
+3. Create a pull request. Use your working branch `<your_initials>_instruct` as the **compare** and use `instructional_review` as the the **base**.
+
+4. Wait for a reviewer to approve your pull request. Address any comments as needed.
+
+5. After you receive approval, merge your pull request.
+
+## Editorial review
+
+### Prerequisite
+
+- You must have an article in the `instructional_review` branch.
+
+### Workflow
+
+You will receive editorial review during this process. Complete the following steps to get your initial draft into the `editorial_review` branch:
+
+1. Create a new working branch. Use `instructional_review` as your **Base**. Name it using the following temple: `<your_initials>_edit`
+
+2. In your working branch, commit any changes you want to your `<article_name>.md` to prepare for editorial review.
+
+4. Create a pull request. Use your working branch `<your_initials>_instruct` as the **compare** and use `instructional_review` as the the **base**.
+
+5. Wait for a reviewer to approve your pull request. Address any comments as needed.
+
+6. After you receive approval, merge your pull request.
+
+## Final draft
+
+### Prerequisite
+
+- You must have an article in the `editorial_review` branch.
+
+### Workflow
+
+You can add your article directly to `final_draft` when it is ready for publishing.
+
+# Microsoft Word workflow
+
+WIP
+
+# Guide for reviewers
+
+WIP
```

Comment:
We should probably discuss some guidelines and list resources for reviewers (style guides, what we expect from our articles etc.)